Constellation/Node.hs

```diff
@@ -128,15 +130,20 @@ sendPayload :: Node
             -> [PublicKey]
             -> IO [Either String Text]
 sendPayload node@Node{..} pl from initRcpts = do
-    let rcpts = nodeAlwaysSendTo ++ initRcpts
+    let (onlySelf, rcpts) = if null rcptsWithAst
```

Comment:
A couple notes: 

* I find the naming a little confusing. Why is it `initRcpts`? What's initial about it? Would `s/initRcpts/explicitRecipients`, `s/rcptsWithAst/allRecipients` make sense?
* We definitely don't want `nodeSelfPub` to always be a recipient?

Reply:
Changed... How's `givenRcpts` ?

No, we definitely don't. It's only to solve for the case where there is no recipient public key otherwise. As long as you have *any* recipient's public key, you can decrypt something you've sent.